urwid/tests/test_vterm.py

```diff
@@ -101,6 +104,7 @@ def resize(self, width: int, height: int, soft: bool = False) -> None:
     def write(self, data: str) -> None:
         data = data.encode('iso8859-1')
         self.command.write(data.replace(br'\e', b'\x1b'))
+        sleep(0.05)
```

Comment:
looks like after write to `stdout` buffer, it's not enough time for full processing and read